server/src/main/java/org/elasticsearch/cluster/metadata/MetaDataCreateIndexService.java

```diff
@@ -367,7 +367,7 @@ public ClusterState execute(ClusterState currentState) throws Exception {
                 // now, put the request settings, so they override templates
                 indexSettingsBuilder.put(request.settings());
                 if (indexSettingsBuilder.get(SETTING_NUMBER_OF_SHARDS) == null) {
-                    indexSettingsBuilder.put(SETTING_NUMBER_OF_SHARDS, settings.getAsInt(SETTING_NUMBER_OF_SHARDS, 5));
+                    indexSettingsBuilder.put(SETTING_NUMBER_OF_SHARDS, settings.getAsInt(SETTING_NUMBER_OF_SHARDS, 1));
```

Comment:
should this be 5 if the index.version.created is < 7.0 I mean we can still have a mixed cluster here no?

Reply:
@s1monw I pushed 2e89d980735b31bb73371eb5f8c37d2ab8c38a34.